Plugin/CsrfValidatorSkip.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Omise\Payment\Plugin;
+use Omise\Payment\Block\Adminhtml\System\Config\Form\Field\Webhook;
+
+class CsrfValidatorSkip
+{
+    protected $urlInterface;
+    public function __construct(\Magento\Framework\UrlInterface $urlInterface) {
```

Comment:
just a little minor on code style. I think we normally have a breakline between function `function` and `{ ... }`